test/kouta_indeksoija_service/queue/queue_test.clj

```diff
@@ -197,6 +197,6 @@
                   (Thread/sleep 1000)
                   (future-cancel f)
                   (combine-messages @handled))))
-            => (contains [{:oid ["expected-123.123.123" "expected-123.123.231" "expected-234.234.234" "expected-234-234-123"]
-                           :boid ["expected-123.123"]
-                           :droid ["expected-234.234"]}])))))
+            => (contains {:oid ["expected-123.123.123" "expected-123.123.231" "expected-234.234.234" "expected-234-234-123"]
```

Comment:
Ilmeisesti tuo midjen päivitys rikkoi tämän, tuli ClassCastException. Kai tämä testaa saman asian?

Reply:
Ehkä. Midjen dokumentaatiosta saa sen käsityksen, että noita []-sulkeita ei tarvita, jos tuo on map. Sitä en osaa sanoa, että miksi niitä ei saisi laittaa ja onko se muuttunut versioiden välillä.

https://github.com/marick/Midje/wiki/Checkers-for-collections-and-strings